src/libs/ReportUtils.ts

```diff
@@ -3995,7 +3898,7 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeEmail, assigneeAccountID
             {
                 onyxMethod: Onyx.METHOD.MERGE,
                 key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${assigneeChatReportID}`,
-                value: {[optimisticAssigneeAddComment.reportAction.reportActionID]: optimisticAssigneeAddComment.reportAction},
+                value: {[optimisticAssigneeAddComment.reportAction.reportActionID ?? '']: optimisticAssigneeAddComment.reportAction},
```

Comment:
```suggestion
                value: {[optimisticAssigneeAddComment.reportAction.reportActionID ?? -1]: optimisticAssigneeAddComment.reportAction},
```

Is possible to use `-1`